ste/mgr-JobPartTransferMgr.go

```diff
@@ -300,6 +301,10 @@ func (jptm *jobPartTransferMgr) FileCountLimiter() common.CacheLimiter {
 // (iii) For completeness, there's also bucket->container name resolution when copying from S3, but that is not expected to ever
 //      create collisions, since it already takes steps to prevent them.
 func (jptm *jobPartTransferMgr) WaitUntilLockDestination(ctx context.Context) error {
+	if strings.EqualFold(jptm.Info().Destination, common.Dev_Null) {
```

Comment:
I couldn't remember clearly, but is this string different on Windows vs Linux/Mac? Maybe double check to make sure.

Reply:
Yes, it is, but our constant is (now) defined as being equal to os.DevNull, so that takes care of it.